HubSpot.NET/Core/HubSpotBaseClient.cs

```diff
@@ -188,7 +189,12 @@ private RestRequest ConfigureRequestAuthentication(string path, Method method)
                         "This authentication mode is no longer supported by hubspot as of November 30, 2020");
                 default:
                     request.AddHeader("Authorization", $"Bearer {_apiKey}");
-                    request.AddHeader("Content-Type", $"application/json");
+
+                    if (!contentType.IsNullOrEmpty())
+                    {
+                        request.AddHeader("Content-Type", contentType);
```

Comment:
allows us to unset the content type. Hubspot doesn't like `application/json` when you're trying to upload a file